globals.go

```diff
@@ -686,6 +688,7 @@ func (r UserRequester) Validate() error {
 		TriggerUserRequester,
 		MergeTestUserRequester,
 		AdHocUserRequester,
+		CreateVersionEndpointUserRequester,
```

Comment:
This is extra but couldn't this just use the AllRequesterTypes and a utility func to do this so we don't have to maintain both?